Examples/Physics_applications/capacitive_discharge/inputs_base_1d_picmi.py

```diff
@@ -98,11 +98,11 @@ def _run_solve(self):
         """Function run on every step to perform the required steps to solve
         Poisson's equation."""
         # get rho from WarpX
-        self.rho_data = fields.RhoFPWrapper(0, False)[...]
+        self.rho_data = fields.RhoFPWrapper(0)[...]
```

Comment:
I would be in favor of introducing an extra layer that makes the ghost cells access more user-friendly, doing something like
```py
        self.rho_data = fields.RhoFP(0, ghost=False)
```
with `RhoFP` defined somewhere like
```py
        def RhoFP(level, ghost):
            if ghost:
                return RhoFPWrapper(level)[()]
            else:
                return RhoFPWrapper(level)[...]
```
hence possibly hiding the distinction of `[()]` vs `[...]`.


@jlvay @RemiLehe @RevathiJambunathan @roelof-groenewald @dpgrote
Sorry for the repeat, but my question would be, why is this option not considered in the same way as the rest of the discussion about keeping the `Wrapper` interfaces for clarity and ease of use?